src/oph/ehoks/db/queries.clj

```diff
@@ -169,6 +169,9 @@
       "heratepalvelu/select_paattyneet_tyoelamajaksot_hyto.sql")
 (defq select-paattyneet-tyoelamajaksot-3kk
       "hoksit/select_paattyneet_tyoelamajaksot_3kk.sql")
+;; TODO: tätä voisi optimoida hajottamalla eri kyselyihin päättyneet
+;; HOKSit ja "unohtuneet" HOKSit jotka etsitään
+;; osaamisenhankkimisjaksojen vanhenemisen perusteella
```

Comment:
Varmaan tiketöitävä?

Reply:
Jeps.  Mulla on tiketissä kommentti tuosta.  Tää on vaan vähän vaikee arvata kun noi query planit ei olleet kauheesti hyödyllisemmät sille "yksinkertaistetulle" kyselylle.

No, mut tiketöin.

Reply:
Nyt https://jira.eduuni.fi/browse/EH-1678